curriculum/challenges/english/14-responsive-web-design-22/learn-css-transforms-by-building-a-penguin/61968e9243a4090cc805531c.md

```diff
@@ -11,10 +11,10 @@ Remove both the horizontal and vertical scrollbars, and prevent programmatic scr
 
 # --hints--
 
-You should give `body` an `overflow` of `clip`.
+You should give `body` an `overflow` of `hidden`.
```

Comment:
This could maybe be a good place to use the placeholders `--fcc-expected--` and `--fcc-actual--` (I think they are like that, it needs to be double checked). `--fcc-actual--` will be the first value in the `assert.equal` statement and `--fcc-expected--` the expected value, so it can be written like
```suggestion
You should give `body` an `overflow` of `--fcc-expected--`. But found `--fcc-actual--`.
```
It can probably be written less awkardly, but that's the idea.